vmdb/app/models/dialog_field_text_box.rb

```diff
@@ -42,6 +42,8 @@ def automate_key_name
   end
 
   def validate(dialog_tab, dialog_group)
+    return if !required? && value.blank?
```

Comment:
The caller reads the return to decide whether the validation has passed, `nil` is the key for the discern. This is existing behavior.